### PR TITLE
add Account Documentation

### PR DESCRIPTION
**Pull request by jancris100**
Docs for sozo account #272 

-New route to account.mdx
-Documentation to account

### CODE REVIEW

File: docs/pages/toolchain/sozo/world-commands/account.mdx [hunk 1]
```diff
@@ -0,0 +1,62 @@
+import WorldOptions from '../common/world-options.mdx'
+import StarknetOptions from '../common/starknet-options.mdx'
+
+## Sozo Account 
+
+The `sozo account` command allows managing accounts for the Sozo platform. 
+It provides functionalities to create new account configurations, deploy account contracts, and fetch 
+existing account configurations.
+## Usage
```
Comment (lambda-0x) [inline docs/pages/toolchain/sozo/world-commands/account.mdx:9]:
```suggestion
The `sozo account` command allows managing account contracts. 
It provides functionalities to create new account configurations, deploying them, and fetch 
account configuration for already existing account.

## Usage
```

**Review by lambda-0x — COMMENTED** [review]
also we should also mention that we only support openzeppelin account variant (for more advance usecases they should use `starkli`) and also add an example.

you can refer to: https://book.starkli.rs/accounts

**Comment by jancris100** [comment]
Oh I haven't read this before I'm already getting to do it

**Comment by jancris100** [comment]
Done the new changes @lambda-0x

File: docs/pages/toolchain/sozo/world-commands/account.mdx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  - `force`: Overwrite the account config file if it already exists.
+  - `file`: Path to save the account config file.
+
+#### `deploy`
+
+Deploy account contract with a `DeployAccount` transaction.
+
+- **Options:**
+  - `starknet`: Options for specifying StarkNet configuration.
+  - `signer`: Options for specifying the signer.
```
Comment (lambda-0x) [inline docs/pages/toolchain/sozo/world-commands/account.mdx:48]: can you please add more description for options that are not directly arguments, something like: https://book.dojoengine.org/toolchain/sozo/world-commands/execute#world-options.

so user can know exactly what argument they need to pass in the CLI.

**Review by lambda-0x — COMMENTED** [review]
one small suggestions, otherwise LGTM.

**Comment by jancris100** [comment]
OK on my way

**Comment by lambda-0x** [comment]
@jancris100 any updates on this?

**Comment by jancris100** [comment]
Hello sorry for the delay I have had very busy weeks today I have time at night and I will update the documentation

**Comment by jancris100** [comment]
Ready the update

**Pull request merged**